src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -412,22 +376,25 @@ def __init__(self, project_id, raw_response):
 
         for hit in raw_response['hits']['hits']:
             if hit['_id'] == project_id:
+                hit_contents = hit['_source']['contents'].to_dict()
                 total_cell_count, organ_cell_count = self.get_cell_count(hit)
+                donor_count = len(self.get_distinct_terms(hit_contents['specimens'], 'donor_biomaterial_id'))
+                species_list = self.get_distinct_terms(hit_contents['specimens'], 'genus_species')
```

Comment:
Eliminating the loop above probably makes the variable redundant. Generally speaking though, unless there is `species_set` and a `species_list` in the same scope, I prefer to not represent the type in the variable name, except to distinguish between single values ("entity") and collections ("entities"). So here that would be just `species`. It's unfortunate that `species` is its own plural but we can gloss over that.